kubernetes/loculus/values.yaml

```
@@ -1665,7 +1665,7 @@ defaultOrganisms:
             sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/cchf/NP.fasta]]"
   enteroviruses:
     <<: *defaultOrganismConfig
-    enabled: false
+    enabled: true
```

Revert before merging
```suggestion
    enabled: false
```